src/Components/Blazor/Server/src/MonoDebugProxy/ws-proxy/DebugStore.cs

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
```

Comment:
Files under `/ws-proxy` need to be excluded from this change as they are shared-source from the Mono repo.

Reply:
Are these files expected to have the MIT header after all changes are made? If so, I recommend keeping them in this state for one commit since the next commit will convert everything seen in this PR to the MIT headers. Otherwise, let me know the expected _final_ state of the files and I'll work on setting things up.

Reply:
No, they are expected to remain identical (byte-for-byte) to their current state.